crates/rolldown/src/module_loader/module_loader.rs

```diff
@@ -323,13 +323,20 @@ impl ModuleLoader {
         if let Some(module) = module.as_normal_mut() {
           // Note: (Compat to rollup)
           // The `dynamic_importers/importers` should be added after `module_parsed` hook.
-          for importer in std::mem::take(&mut self.intermediate_normal_modules.importers[id]) {
+          let importers = std::mem::take(&mut self.intermediate_normal_modules.importers[id]);
+          for importer in &importers {
             if importer.kind.is_static() {
-              module.importers.push(importer.importer_path);
+              module.importers.push(importer.importer_path.clone());
             } else {
-              module.dynamic_importers.push(importer.importer_path);
+              module.dynamic_importers.push(importer.importer_path.clone());
             }
           }
+          if !importers.is_empty() {
```

Comment:
This step could be parallel

Reply:
not a blocker, if the benchmark shows no regression, I am ok with it.

Reply:
Yeah. I'm doing it.

Reply:
The step run at parallel is not showing performance improve at benchmark, the start thread is more has overhead at here.

Reply:
That's alright, we could do it when it shows regression.